lib/sisimai/bite/email/mxlogic.rb

```diff
@@ -0,0 +1,253 @@
+module Sisimai::Bite::Email
+  module MXLogic
+    # Sisimai::Bite::Email::MXLogic parses a bounce email which created by
+    # McAfee SaaS (formerly MX Logic).
+    # Methods in the module are called from only Sisimai::Message.
+    class << self
+      # Imported from p5-Sisimail/lib/Sisimai/Bite/Email/MXLogic.pm
+      # Based on Sisimai::Bite::Email::Exim
+      require 'sisimai/bite/email'
+
+      Re0 = {
+        :'from'      => %r/\AMail Delivery System/,
+        :'subject'   => %r{(?:
+             Mail[ ]delivery[ ]failed(:[ ]returning[ ]message[ ]to[ ]sender)?
+            |Warning:[ ]message[ ].+[ ]delayed[ ]+
+            |Delivery[ ]Status[ ]Notification
+            )
+        }x,
+        :'message-id' => %r/\A[<]mxl[~][0-9a-f]+/,
+      }
+      Re1 = {
+        :rfc822 => %r/\AIncluded is a copy of the message header:\z/,
+        :begin  => %r/\AThis message was created automatically by mail delivery software[.]\z/,
+        :endof  => %r/\A__END_OF_EMAIL_MESSAGE__\z/,
+      }
+      ReCommand = [
+        %r/SMTP error from remote (?:mail server|mailer) after ([A-Za-z]{4})/,
+        %r/SMTP error from remote (?:mail server|mailer) after end of ([A-Za-z]{4})/,
+      ]
+      ReFailure = {
```

Comment:
__[rubocop]__
* Style/MutableConstant: Freeze mutable objects assigned to constants.

<!-- Comment by [SideCI](https://sideci.com) -->